### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AlexGrytsai**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+        return self
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```
Comment (vasylhnatiuk) [inline app/main.py:17]:
```suggestion
        self.skills += ["JavaScript", "HTML", "CSS"]
```

Comment (vasylhnatiuk) [reply app/main.py:17]: you must add to previous skills

File: app/main.py [hunk 2]
```diff
@@ -1 +1,53 @@
...
+        super().__init__(name)
+        self.skills = ["Java", "Android studio"]
+
+    def create_smooth_mobile_app(self) -> str:
+        print(f"{self.name} is creating a mobile app...")
+        return "Ads every three swipes"
+
+
+class FullStackDeveloper(BackendDeveloper, FrontendDeveloper):
+    def __init__(self, name: str) -> None:
```
Comment (vasylhnatiuk) [inline app/main.py:45]: if you use += you can avoid `init` here

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
.

**Review by YukihiroSM — APPROVED** [review]
Looks good! Also, note, that you can use something like: self.skills.extend() to extend existing list with some values.

**Pull request closed without merging**